src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureBlobAction.java

```diff
@@ -19,10 +19,9 @@
 import java.util.List;
 
 @ExportedBean
-public class AzureBlobAction implements RunAction {
```

Comment:
`RunAction` is deprecated. In the original implementation, we can actually remove it completely, as we didn't make use of this (broken) interface at all.

Migrate to use `RunAction2` so we don't need to store duplicated build information.